src/engines/synths/OTTOFM/audio.hpp

```diff
@@ -142,7 +159,12 @@ namespace otto::engines::ottofm {
     float previous_value_ = 0;
   };
 
-  struct Voice : voices::VoiceBase<Voice> {
+  struct Voice : voices::VoiceBase<Voice>,
+                 itc::ActionReceiverOnBus<itc::AudioBus,
+                                          voices::attack_tag::action,
+                                          voices::decay_tag::action,
+                                          voices::sustain_tag::action,
+                                          voices::release_tag::action> {
     Voice(Audio& a) noexcept;
```

Comment:
Voice needs to add operators as children from its constructor.
Also, the voice manager needs to register the voices as children, which is slightly more involved, since it needs to be conditional based on whether or not the voice actually extends `ActionReceiverOnBusBase`